test/bigtest/tests/utils-test.js

```diff
@@ -91,6 +91,7 @@ describe('utils', () => {
       firstName: 'Steven',
       lastName: 'Jones',
       middleName: 'Jacob',
+      preferredFirstName: 'Paul',
```

Comment:
Can we add 1 more test for check correct behavior when `preferredFirstName  `are absent?